### PR TITLE
Build on windows

### PR DESCRIPTION
**Pull request by albfan**
Add support for windows.

Working on mingw to run Akira:

https://github.com/msys2/MINGW-packages/pull/8505/files#diff-18b1226b74e8b4adf04b5e3401ac6886ead2da9a1dc26b6c61be29141ca0f641R23

### CODE REVIEW

File: lib/Application.vala [hunk 1]
```diff
@@ -154,8 +154,10 @@ namespace Granite {
             Granite.Services.Logger.initialize (program_name);
             Granite.Services.Logger.DisplayLevel = Granite.Services.LogLevel.INFO;
             message ("%s version: %s", program_name, build_version);
+#if ! WINDOWS
```
Comment (tintou) [inline lib/Application.vala:157]:
```suggestion
#if !WINDOWS
```

File: lib/Services/System.vala [hunk 2]
```diff
@@ -144,9 +144,12 @@ namespace Granite.Services {
                 return;
 
             AppInfo info;
+
+#if ! WINDOWS
```
Comment (tintou) [inline lib/Services/System.vala:148]:
```suggestion
#if !WINDOWS
```

File: lib/Application.vala (hunk 1 above)
Comment (albfan) [reply lib/Application.vala:157]: 😉

File: lib/Services/System.vala (hunk 2 above)
Comment (albfan) [reply lib/Services/System.vala]: Done

**Comment by tintou** [comment]
It looks good to me, Although I haven't tested it. I don't see why we couldn't merge this as it's painless for us

**Review by tintou — APPROVED** [review]

**Pull request merged**